sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -38,9 +38,9 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder} EXCLUDE_FROM_ALL)
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.3.1" CONFIG QUIET)
+  find_package(azure-core-cpp "1.8.0" CONFIG QUIET)
```

Comment:
Is this the **minimum** version of core that identity now depends on? Will anything older cause it to break?

Reply:
Yes, it needs the new Core version, In Core's changelog for "1.8.0", there is an entry under "Other Changes": "Added support for Identity token caching, and for configuring token refresh offset in `BearerTokenAuthenticationPolicy`."